fdbclient/include/fdbclient/CommitProxyInterface.h

```diff
@@ -123,7 +123,6 @@ struct ClientDBInfo {
 	Optional<Value> forward;
 	std::vector<VersionHistory> history;
 	UID clusterId;
-	bool isEncryptionEnabled = false;
```

Comment:
This flag is already on release-7.2 and snowflake/release-71.2 branches. I'm wondering what's the correct way to deprecate a field in flatbuffer struct.

Reply:
discussed offline and the conclusion was it is safe to remove a field as long as it is not persisted and not included in a stable endpoint (i.e. the endpoint to get the protocol version).